public/locales/en/status.json

```diff
@@ -1,27 +1,27 @@
 {
   "date-of-birth": {
     "error": {
-      "current": "The $t(date-of-birth.label) $t(common:error-messages.in-past)",
-      "invalid": "The $t(date-of-birth.label) $t(common:error-messages.valid-format, { \"format\": \"dd/mm/yyyy\" })",
-      "required": "The $t(date-of-birth.label) $t(common:error-messages.required)"
+      "current": "The date of birth must be in the past.",
+      "invalid": "The date of birth must be in a valid format of «\u00a0yyyy-MM-dd\u00a0».",
```

Comment:
Aren't «» only used as quotations in French? Should this not be use `""` here?

Reply:
You might be right, thanks.

Will adjust and will see what is done on Canada.ca websites.

Reply:
Adjusted based on an IRCC guide:

- [English](https://www.canada.ca/en/immigration-refugees-citizenship/services/application/application-forms-guides/guide-5218-request-amend-record-landing-confirmation-permanent-residence-valid-temporary-resident-documents.html#:~:text=date%20of%20birth-,in%20the%20following%20format%3A%20YYYY%2DMM%2DDD%20(Year%2DMonth%2DDay),-Question%208)
- [French](https://www.canada.ca/fr/immigration-refugies-citoyennete/services/demande/formulaires-demande-guides/guide-5218-demande-modification-fiche-relative-droit-etablissement-confirmation-residence-permanente-documents-resident-temporaire-valides.html#:~:text=date%20de%20naissance-,selon%20le%20format%20suivant%C2%A0%3A%20AAAA%2DMM%2DJJ%20(Ann%C3%A9e%2DMois%2DJour).,-Question%208)